src/components/RegistrationList.tsx

```diff
@@ -223,6 +226,16 @@ export const RegistrationListItemContent = ({
           )}
         </Box>
       )}
+      {onRemoveRelated && (
+        <Tooltip title={t('registration.resource_type.research_data.remove_relation')}>
+          <IconButton
+            sx={{ alignSelf: 'start' }}
+            onClick={onRemoveRelated}
+            data-testid={dataTestId.registrationWizard.resourceType.removeRelationButton(registration.identifier)}>
+            <CloseIcon sx={{ bgcolor: 'primary.main', color: 'white', borderRadius: '50%', p: '0.25rem' }} />
```

Comment:
liker nok å bruke filled ikon bedre, siden det er både enklere og vi vet at knappen skalerer riktig og ikke blir litt off pga padding. Samme med prosjekt også. Men de gjør stort sett samme nytten, så opp til deg!
(samme med prosjekt)
```suggestion
            <CancelSharpIcon color="primary" />
```